Cubical/Experiments/CountingFiniteStructure.agda

```diff
@@ -1,116 +0,0 @@
-{-
```

Comment:
This I want to keep. Can you make it so that it gets checked when checking Experiments.Everything?

Reply:
This checks

Reply:
Added to Everything